featureHome/src/main/java/com/tapptitude/featurehome/presentation/HomeViewModel.kt

```diff
@@ -16,7 +16,7 @@ class HomeViewModel(
     private val loadImageUseCase: LoadImageUseCase,
     private val sessionManager: SessionManager
 ) : ViewModel() {
-    private val _isLoading = MutableLiveData<Boolean>()
+    private val _isLoading = MutableLiveData<Boolean>(false)
     val isLoading: LiveData<Boolean> = _isLoading
```

Comment:
I think this is supposed to have a getter instead of assigned MutableLiveData value, right?